src/org/sosy_lab/java_smt/test/InterpolatingProverTest.java

```diff
@@ -282,31 +284,31 @@ public <T> void binaryBVInterpolation1() throws SolverException, InterruptedExce
 
   private void requireTreeItp() {
     requireInterpolation();
-    if (interpolationStrategy == null) {
-      assume()
-          .withMessage("Solver does not support tree-interpolation.")
-          .that(solver)
-          .isAnyOf(Solvers.SMTINTERPOL, Solvers.PRINCESS);
-    } else {
-      assume()
-          .withMessage(
-              "Tree interpolants are not supported for independent interpolation currently.")
-          .that(interpolationStrategy)
-          .isAnyOf(
-              ProverOptions.GENERATE_MODEL_BASED_INTERPOLANTS,
-              ProverOptions.GENERATE_UNIFORM_FORWARD_INTERPOLANTS,
-              ProverOptions.GENERATE_UNIFORM_FORWARD_INTERPOLANTS);
-    }
+    assume()
+        .withMessage("Solver does not support tree-interpolation.")
+        .that(solver)
+        .isAnyOf(Solvers.SMTINTERPOL, Solvers.PRINCESS);
+
+    assume()
+        .withMessage("Independent interpolation strategies do not support tree-interpolation.")
+        .that(interpolationStrategy)
+        .isAnyOf(ProverOptions.GENERATE_MODEL_BASED_INTERPOLANTS,
+            ProverOptions.GENERATE_UNIFORM_FORWARD_INTERPOLANTS,
+            ProverOptions.GENERATE_UNIFORM_BACKWARD_INTERPOLANTS);
   }
 
-  private void requireIndependentSeqItp() {
+  private void requireSequentialInterpolation() {
     requireInterpolation();
+    assume()
+        .withMessage("Solver does not support tree-interpolation.")
```

Comment:
Either the text is wrong, or you are confusing tree and sequential interpolants.